examples/led_light/main/app_main.c

```diff
@@ -34,26 +34,23 @@ extern const char ota_server_cert[] asm("_binary_server_crt_start");
 static esp_err_t write_cb(const esp_rmaker_device_t *device, const esp_rmaker_param_t *param,
             const esp_rmaker_param_val_t val, void *priv_data, esp_rmaker_write_ctx_t *ctx)
 {
-    if (ctx) {
-        ESP_LOGI(TAG, "Received write request via : %s", esp_rmaker_device_cb_src_to_str(ctx->src));
+    if (ctx)
+    {
+        ESP_LOGI(TAG, "Received write request via context '%s'",
+                 esp_rmaker_device_cb_src_to_str(ctx->src));
     }
+
+    // Values are logged from built-in tag esp_rmaker_param
```

Comment:
Sorry for the delay in review, but these prints are actually helpful to debug issues. The prints from SDK are the raw data sent by clients like mobile phones (or scenes/schedules), whereas the prints here mean that the values could be parsed and mapped to appropriate devices and params. We have indeed come across cases wherein people had been passing wrong values (Eg. "brightness":"55" instead of "brightness":55), wherein these prints helped.